src/common/reader.js

```diff
@@ -965,6 +976,32 @@ class Reader {
 		document.getElementById("a11yAnnouncement").innerText = a11yMessage;
 	}
 
+	// Make a11yVirtualCursorTarget node set previously focusable and
+	// focus it to help screen readers understand where the virtual cursor needs to
+	// be positioned. This is required because screen readers are not aware of
+	// scroll positioning, so without this, the virtual cursor will always land
+	// at the start of the document.
+	placeA11yVirtualCursor () {
+		let target = this._state.a11yVirtualCursorTarget;
+		let doc = this._lastView._iframe.contentDocument;
+		// If the target is a text node, use its parent (e.g. <p> or <h>)
+		if (target?.nodeName == "#text") {
+			target = target.parentNode;
+		}
+		if (!target || !doc.contains(target)) return;
```

Comment:
It's certainly not intended but I think there can be some odd instances of when it would happen without something completely breaking... 

For example, if one uses the split horizontal or vertical views. One can navigate the outline, which will remember `a11yVirtualCursorTarget` for one view, then hit shift-tab a bunch of times to wrap focus around so it enters the other view. That other view would not have that node from the first view. Keeping in mind that this is to improve the experience of screen reader users, this example is likely not very appropriate but still something I thought of.